src/5.0_to_5.1/__test_resources__/smallContentServer.ts

```diff
@@ -0,0 +1,1240 @@
+                    lastEditor: "admin",
+                    canRead: true,
+                    canWrite: true,
+                  },
+                },
+                "4a8": {
+                  entry: {
+                    content:
+                      '{"expression":"Log([Measures].[pv.SUM], 10)","properties":["FORMAT_STRING = \\"#,###.##\\""]}',
+                    isDirectory: false,
+                    owners: ["admin"],
+                    readers: ["admin"],
+                    timestamp: 1669805699068,
+                    lastEditor: "admin",
+                    canRead: true,
+                    canWrite: true,
+                  },
+                },
+              },
+            },
+            structure: {
+              entry: {
+                isDirectory: true,
+                owners: ["ROLE_USER"],
+                readers: ["ROLE_USER"],
+                timestamp: 1669226473767,
+                lastEditor: "_no_user_",
+                canRead: true,
+                canWrite: true,
+              },
+              children: {
+                "1d3": {
+                  entry: {
+                    isDirectory: true,
+                    owners: ["admin"],
+                    readers: ["admin"],
+                    timestamp: 1669827289319,
+                    lastEditor: "admin",
+                    canRead: true,
+                    canWrite: true,
+                  },
+                  children: {
+                    "1d3_metadata": {
+                      entry: {
+                        content: '{"name":"CM in 2 cubes"}',
+                        isDirectory: false,
+                        owners: ["admin"],
+                        readers: ["admin"],
+                        timestamp: 1669827289319,
+                        lastEditor: "admin",
+                        canRead: true,
+                        canWrite: true,
+                      },
+                    },
+                  },
+                },
+                "4a8": {
+                  entry: {
+                    isDirectory: true,
+                    owners: ["admin"],
+                    readers: ["admin"],
+                    timestamp: 1669805699247,
+                    lastEditor: "admin",
+                    canRead: true,
+                    canWrite: true,
+                  },
+                  children: {
+                    "4a8_metadata": {
+                      entry: {
+                        content: '{"name":"Log pv.SUM"}',
+                        isDirectory: false,
+                        owners: ["admin"],
+                        readers: ["admin"],
+                        timestamp: 1669805699247,
+                        lastEditor: "admin",
+                        canRead: true,
+                        canWrite: true,
+                      },
+                    },
+                  },
+                },
+                aca: {
```

Comment:
I see `aca` and `9a7` in `structure` but not in `content`, even though as their metadata file suggests they're not folders.
Let's adapt this.
And actually, it would be nice to have a calculated measure under a folder.

Reply:
Should be good now!